server/src/test/java/com/broadcom/lsp/cobol/usecases/TestXmlQueryNotTerminatesSemanticAnalysis.java

```diff
@@ -42,6 +35,29 @@
  * semantic analyzer could not find a token for variable usages, producing unexpected behavior.
  */
 class TestXmlQueryNotTerminatesSemanticAnalysis {
+  // TODO check if below commented syntax is valid.
```

Comment:
The SQL here was cut. The original code example is in the INSXML.cbl

Reply:
I would kindly ask you to revert the changes in this test and make it ignored. It requires additional investigation.

Reply:
@temanbrcom Yes, done. Now all requested-changes are done (except this test)